test/io/inputs.jl

```diff
@@ -84,16 +85,16 @@ end
         "Incorrect dimension name. Expected $(name(dim_1)) \
          but received $(name(yax_res.axes[1]))"
 
-    @test dim_1_vals == collect(yax_res.axes[1]) || 
+    @test dim_1_vals == collect(yax_res.axes[1]) ||
         "Incorrect axis indices. Expected $(dim_1_vals) but received $(collect(yax_res.axes[1]))"
-      
+
         yax_res = ZeroDataCube(; T=Float64, NamedTuple{(dim_1_name, dim_2_name)}((dim_1_vals, dim_2_vals))...)
 
-    @test typeof(yax_res) <: YAXArray{Float64, 2} || 
+    @test typeof(yax_res) <: YAXArray{Float64, 2} ||
```

Comment:
Sorry it was this one I meant to mark. It tests for Float64 but message says Int

Reply:
Well, yes, in this case the message is wrong. (:

Reply:
Thanks, wasn't sure if the test was wrong or the message